sonoff/webserver.ino

```diff
@@ -841,7 +843,6 @@ void HandleWifi(boolean scan)
   }
 
   page += FPSTR(HTTP_FORM_WIFI);
-  page.replace(F("{h1"), Settings.hostname);
   page.replace(F("{s1"), Settings.sta_ssid[0]);
```

Comment:
It is missing the line:

`page.replace(F("{h1"), Settings.hostname);`

Should be there because `Settings.hostname` is different than `my_hostname`.

Reply:
Ah, right, I misunderstood, my bad